articles/anomaly-detection/references/attack-protection-triggers-actions.md

```diff
@@ -51,14 +53,51 @@ If this block is triggered, additional access attempts are released one-at-a-tim
 
 ### Remove block
 
-Auth0 emails the dashboard administrator(s) when this block is triggered. The email contains a link that the owner can click to navigate to tenant logs to examine which IPs have been blocked. Recent blocks can be found using this query:
-```
+Auth0 emails the Dashboard administrator(s) when this block is triggered. The email contains a link that the owner can click to navigate to tenant logs to examine which IPs have been blocked. 
+
+Recent blocks can be found using this query:
+
+```text
 type:limit_mu
 ```
+
 Blocks can then be removed using the [Management API](/api/management/v2#!/Anomaly/delete_ips_by_id).
 
+## Credential stuffing attacks
+
+### Trigger
+
+A single host (IP) which attempts more than 100 distinct username/password combos over a sliding 24 hour window with less than 1% of successful logins. This is considered a credential stuffing attack. 
```

Comment:
Do we want to expose this level of internal detail?  Suppose customers write monitoring systems or test harnesses with this criteria, then we change it (and they break)?  Would ideally like to give a description of *what* it does, and less of the exact evaluation criteria.

Reply:
Axe all this stuff